unblob/processing.py

```diff
@@ -409,6 +409,12 @@ def _extract_chunk(self, file, chunk: ValidChunk):
             )
 
 
+def delete_empty_extract_dir(extract_dir: Path):
+    # delete empty extract directories
+    if extract_dir.exists() and next(extract_dir.iterdir(), None) is None:
```

Comment:
easier is:

`if extract_dir.exists() and not any(extract_dir.iterdir()):`

or just use `rmdir` and handle `OSError: [Errno 39] Directory not empty` error

Reply:
Used your construct and removed the superfluous comment.